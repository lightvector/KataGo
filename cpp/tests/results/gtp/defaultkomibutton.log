: Running with following config:
allowResignation = true
cudaUseFP16 = false
cudaUseNHWC = false
forDeterministicTesting = true
hasButton = true
koRule = POSITIONAL
lagBuffer = 1.0
logAllGTPCommunication = true
logFile = tests/results/gtp/defaultkomibutton.log
logSearchInfo = true
logSearchInfoForChosenMove = false
logTimeStamp = false
logToStderr = false
maxPlayouts = 10000
maxTimePondering = 60.0
maxVisits = 100
multiStoneSuicideLegal = false
nnRandSeed = forTesting
nnRandomize = false
numSearchThreads = 1
openclUseFP16 = false
ponderingEnabled = false
resignConsecTurns = 3
resignThreshold = -0.90
rootSymmetryPruning = false
scoringRule = AREA
searchFactorAfterOnePass = 0.50
searchFactorAfterTwoPass = 0.25
searchFactorWhenWinning = 0.40
searchFactorWhenWinningThreshold = 0.95
searchRandSeed = forTesting
taxRule = NONE
trtUseFP16 = false

: GTP Engine starting...
: KataGo v1.16.4
: Using koPOSITIONALscoreAREAtaxNONEsui0button1 rules initially, unless GTP/GUI overrides this
: Using 1 CPU thread(s) for search
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 19 * 19 exactly
: Cuda backend thread 0: Found GPU NVIDIA RTX A5000 memory 25425608704 compute capability major 8 minor 6
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 19 nnYLen 19
: Initializing board with boardXSize 19 boardYSize 19
: Loaded config configs/gtp_example.cfg and/or command-line and query overrides
: Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: Config override: cudaUseFP16 = false
: Config override: cudaUseNHWC = false
: Config override: forDeterministicTesting = true
: Config override: hasButton = true
: Config override: koRule = POSITIONAL
: Config override: logDir = 
: Config override: logFile = tests/results/gtp/defaultkomibutton.log
: Config override: logTimeStamp = false
: Config override: maxPlayouts = 10000
: Config override: maxVisits = 100
: Config override: multiStoneSuicideLegal = false
: Config override: nnRandSeed = forTesting
: Config override: nnRandomize = false
: Config override: numSearchThreads = 1
: Config override: openclUseFP16 = false
: Config override: rootSymmetryPruning = false
: Config override: rules = 
: Config override: scoringRule = AREA
: Config override: searchRandSeed = forTesting
: Config override: taxRule = NONE
: Config override: trtUseFP16 = false
: Model name: g170-b6c96-s175395328-d26788732
: GTP ready, beginning main protocol loop
: Controller: genmove_debug b
: MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . @ . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui0button1komi7
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 82
NN batches: 82
NN avg batch size: 1
PV: Q17 D3 C16 Q3 Q15
Tree:
: T   0.96c W   0.73c S   0.23c ( +0.3 L  +0.4) N     100  --  Q17 D3 C16 Q3 Q15
---Black(^)---
Q17 : T   1.95c W   1.24c S   0.35c ( +0.5 L  +0.5) LCB   -1.60c P  8.95% WF  17.1 PSV      21 N      17  --  Q17 D3 C16 Q3 Q15
D3  : T   1.54c W   1.23c S   0.35c ( +0.5 L  +0.6) LCB   -2.92c P  8.50% WF  14.0 PSV      13 N      14  --  D3 R16 C16 Q3 E16
Q3  : T   1.16c W   1.19c S   0.30c ( +0.4 L  +0.5) LCB   -4.26c P  9.03% WF  12.0 PSV      11 N      12  --  Q3 D3 C16 Q17 E16
C16 : T   1.19c W   0.90c S   0.26c ( +0.4 L  +0.3) LCB   -5.13c P  7.99% WF  12.0 PSV      10 N      12  --  C16 R16 D3 Q3 E16
R16 : T   1.01c W   0.86c S   0.25c ( +0.4 L  +0.4) LCB   -5.67c P  7.51% WF  11.0 PSV       9 N      11  --  R16 D3 C16 Q3
R4  : T   0.60c W   0.44c S   0.13c ( +0.2 L  +0.2) LCB   -8.44c P  9.08% WF   9.0 PSV       9 N       9  --  R4 D3 C16 Q17 E16
D17 : T   0.58c W   0.20c S   0.13c ( +0.2 L  +0.2) LCB  -10.92c P  8.03% WF   8.0 PSV       8 N       8  --  D17 R16 D3 Q3 D5
C4  : T  -0.03c W   0.28c S   0.12c ( +0.2 L  +0.2) LCB  -15.42c P  8.65% WF   7.0 PSV       7 N       7  --  C4 R16 C16 Q3
Q4  : T  -0.26c W  -0.23c S   0.11c ( +0.2 L  +0.2) LCB  -43.19c P  5.57% WF   4.0 PSV       4 N       4  --  Q4 D3 D17 Q17
D4  : T   0.10c W   0.21c S  -0.06c ( -0.1 L  -0.1) LCB  -73.92c P  4.56% WF   3.0 PSV       3 N       3  --  D4 R16 D17

: = Q3
: GPU -1 finishing, processed 82 rows 82 batches
: All cleaned up, quitting
