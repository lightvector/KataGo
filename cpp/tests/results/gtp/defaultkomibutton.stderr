KataGo v1.8.0
Using koPOSITIONALscoreAREAtaxNONEsui0button1 rules initially, unless GTP/GUI overrides this
Loaded config configs/gtp_example.cfg and/or command-line and query overrides
Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
Model name: g170-b6c96-s175395328-d26788732
GTP ready, beginning main protocol loop
MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . @ . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui0button1komi7
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 95
NN batches: 95
NN avg batch size: 1
PV: D3 R16 D17 Q3
Tree:
: T   0.69c W   0.05c S   0.64c ( +0.2 L  +0.2) N     100  --  D3 R16 D17 Q3
---Black(^)---
D3  : T   1.86c W   0.92c S   0.94c ( +0.4 L  +0.5) LCB   -7.05c P  8.50% WF  6.82% PSV      16 N      10  --  D3 R16 D17 Q3
Q3  : T   1.29c W   0.59c S   0.71c ( +0.4 L  +0.6) LCB   -7.87c P  9.03% WF  6.76% PSV      10 N      10  --  Q3 R16 D3 D17
Q17 : T   1.41c W   0.66c S   0.75c ( +0.3 L  +0.3) LCB   -8.32c P  8.96% WF  6.78% PSV      10 N      10  --  Q17 D3 D17 Q3
C16 : T   2.33c W   1.03c S   1.30c ( +0.3 L  +0.3) LCB   -7.51c P  7.98% WF  6.87% PSV      10 N      10  --  C16 D3 Q17 Q3
R4  : T   0.40c W  -0.31c S   0.71c ( +0.1 L  +0.2) LCB  -11.24c P  9.07% WF  6.67% PSV       9 N       9  --  R4 R16 D3 D17
C4  : T   0.43c W  -0.26c S   0.69c ( +0.1 L  +0.2) LCB  -12.74c P  8.66% WF  6.67% PSV       9 N       9  --  C4 R16 D17 Q3
R16 : T   1.28c W   0.46c S   0.83c ( +0.3 L  +0.4) LCB  -10.00c P  7.52% WF  6.76% PSV       9 N       9  --  R16 D3 D17 Q3
D17 : T   0.48c W  -0.35c S   0.83c ( +0.1 L  +0.2) LCB  -12.62c P  8.03% WF  6.68% PSV       8 N       9  --  D17 D3 Q17 Q3
Q4  : T  -0.31c W  -0.36c S   0.05c ( +0.0 L  +0.1) LCB  -25.70c P  5.57% WF  6.61% PSV       5 N       5  --  Q4 D3 D17
D16 : T  -1.26c W  -1.35c S   0.09c ( -0.3 L  -0.2) LCB -104.87c P  4.57% WF  6.53% PSV       4 N       4  --  D16 D3 Q17
